doc/cascadia/profiles.schema.json

```diff
@@ -0,0 +1,449 @@
+          "cursorColor": {
+            "$ref": "#/definitions/Color",
+            "default": "#FFFFFF",
+            "description": "Sets the cursor color for the profile. Uses hex color format: \"#rrggbb\"."
+          },
+          "cursorHeight": {
+            "description": "Sets the percentage height of the cursor starting from the bottom. Only works when cursorShape is set to \"vintage\". Accepts values from 25-100.",
+            "maximum": 100,
+            "minimum": 25,
+            "type": "integer"
+          },
+          "cursorShape": {
+            "default": "bar",
+            "description": "Sets the cursor shape for the profile. Possible values: \"vintage\" ( ▃ ), \"bar\" ( ┃, default ), \"underscore\" ( ▁ ), \"filledBox\" ( █ ), \"emptyBox\" ( ▯ )",
+            "enum": [
+              "bar",
+              "emptyBox",
+              "filledBox",
+              "underscore",
+              "vintage"
+            ],
+            "type": "string"
+          },
+          "fontFace": {
+            "default": "Consolas",
+            "description": "Name of the font face used in the profile.",
+            "type": "string"
+          },
+          "fontSize": {
+            "default": 12,
+            "description": "Sets the font size.",
+            "minimum": 1,
+            "type": "integer"
+          },
+          "foreground": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the foreground color of the profile. Overrides foreground set in color scheme if colorscheme is set. Uses hex color format: \"#rrggbb\". Default \"#ffffff\" (white)."
+          },
+          "guid": {
+            "$ref": "#/definitions/ProfileKey",
+            "description": "Unique identifier of the profile. Written in registry format: \"{00000000-0000-0000-0000-000000000000}\"."
+          },
+          "historySize": {
+            "default": 9001,
+            "description": "The number of lines above the ones displayed in the window you can scroll back to.",
+            "minimum": -1,
+            "type": "integer"
+          },
+          "icon": {
+            "description": "Image file location of the icon used in the profile. Displays within the tab and the dropdown menu.",
+            "type": "string"
+          },
+          "name": {
+            "default": "PowerShell Core",
+            "description": "Name of the profile. Displays in the dropdown menu.",
+            "minLength": 1,
+            "type": "string"
+          },
+          "padding": {
+            "default": "8, 8, 8, 8",
+            "description": "Sets the padding around the text within the window. Can have three different formats: \"#\" sets the same padding for all sides, \"#, #\" sets the same padding for left-right and top-bottom, and \"#, #, #, #\" sets the padding individually for left, top, right, and bottom.",
+            "pattern": "^-?[0-9]+(\\.[0-9]+)?( *, *-?[0-9]+(\\.[0-9]+)?|( *, *-?[0-9]+(\\.[0-9]+)?){3})?$",
```

Comment:
thanks, i hate this.

Reply:
I tried your suggestion and the profiles.json didn't like it with the proper padding syntax :(